Step_Functions/Preprocessing_All/Resampling.m

```diff
@@ -40,6 +40,8 @@
 % Order determines when it should be run.
 StepName = "Resampling";
 Choices = ["500", "250", "125"];
+% the preregistration states that data is downsampled by factors, so would differ depending on amplifier
+% so would be 512/500, 256/250, 128/125 Hz
```

Comment:
as you can see below (line 110) in the script it is kept relative to the original sampling rate. This is just for "naming" and executing - so please remove or answer before merging/pull request